packages/config/src/chainConfig/types.ts

```diff
@@ -49,11 +49,18 @@ export type ChainConfig = {
   GLOAS_FORK_EPOCH: number;
 
   // Time parameters
-  SECONDS_PER_SLOT: number;
+  SECONDS_PER_SLOT: number; // Deprecated
```

Comment:
might be better to do

```suggestion
  /** @deprecated */
  SECONDS_PER_SLOT: number;
```